src/site/content/en/blog/pwas-on-oculus-2/index.md

```diff
@@ -296,66 +296,136 @@ customize the PWA experience.
 </table>
 </div>
 
+### Packaging PWAs with Bubblewrap CLI
+
+[Bubblewrap](https://github.com/GoogleChromeLabs/bubblewrap) is an open source set of libraries and
+a command line tool (CLI) for Node.js developed by the Google Chrome team that helps developers
+generate, build, and sign an Android project that launches your PWA as a
+[Trusted Web Activity](https://developer.chrome.com/docs/android/trusted-web-activity/) (TWA).
+
+Meta Quest Browser currently doesn't fully support TWA, but starting from version 1.18.0,
+Bubblewrap [supports](https://github.com/GoogleChromeLabs/bubblewrap/releases/tag/v1.18.0) packaging
+PWAs for Meta Quest devices.
+
+It can generate *universal* APKs that open a TWA on regular Android devices and open Meta Quest
+Browser on Meta Quest devices, so it can be published to the Google Play Store and the Meta Quest Store.
```

Comment:
```suggestion
It can generate *universal* APKs that open a TWA on regular Android devices and Meta Quest
Browser on Meta Quest devices, so the APK can be published to the Google Play Store and the Meta Quest Store.
```

Reply:
I think this is right, it wasn't clear what "it" referred to.

Reply:
@rachelandrew Thanks! Applied. :)